coralproject/talk
# [CORL-3062]: Update notifications copy and design

## What does this PR do?

These changes update the copy and design for notifications in the feed for replies, featured comments, approvals of comments, etc.

Includes:
- [x] Update copy/styles for replies (staff and commenter) notifications
- [x] Update copy/styles for featured comments notifications
- [x] Update copy/styles for approvals of comments
- [x] Add in support for media in comments in notifications feeds
- [x] Add note about adjusting user preferences at top of notifications that links to `My Profile` > `Preferences`
- [x] Updates so that featured comments don't also create an approved comment notification

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags?

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Reply to comments as regular comments and staff members. See notifications in feed. Feature comments. See notifications in feed. Approve comments as moderator. See notifications in feed. See that the notifications in the feed match designs.

Add media to comments and see that the media shows in the comment notifications feed.

Click on the link about adjusting user preferences at the top of the notifications feed and see that it goes to `My Profile` > `Preferences`

## Were any tests migrated to React Testing Library?



## How do we deploy this PR?
